ktor-retrofit/src/main/kotlin/com/bnorm/ktor/retrofit/RetrofitService.kt

```diff
@@ -170,6 +182,20 @@ private fun Route.process(service: Any, function: KFunction<*>) {
   TODO("implement the rest of the function annotations : $annotations")
 }
 
+private fun checkRegisteredRoute(
+  method: HttpMethod,
+  baseUrl: Route,
+  path: String,
+  methodName: String
+) {
+  val newRoute = RegisteredRoute(method, baseUrl.toString(), path, methodName)
+  val registeredPaths = registeredRoutes.getOrDefault(method, mutableSetOf())
+  if (registeredPaths.contains(newRoute)) {
+    throw IllegalStateException("@${method.value}(\"$path\") is already registered")
```

Comment:
I think it might be helpful to include the names of the interfaces that are conflicting. If it's not convenient, we can leave it as a future enhancement, but I think that information would be very helpful when debugging this error.

Reply:
Yeah, I thought so too. Added